cmd/commands/dr_health.go

```diff
@@ -0,0 +1,28 @@
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/dr"
+	"github.com/spf13/cobra"
+)
+
+var DrCmd = &cobra.Command{
+	Use:                "dr",
+	Short:              "Calls subcommand health",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
+}
+
+var healthCmd = &cobra.Command{
+	Use:                "health",
+	Short:              "Print the ceph status of a peer cluster in a mirroring-enabled cluster. Optional ceph args such as: '--debug-ms 1'.",
```

Comment:
It seems odd to include the optional arg in the command description, they should be able to discovery the args in another place
```suggestion
	Short:              "Print the ceph status of a peer cluster in a mirroring-enabled cluster,
```

Reply:
done